src/modelfree/shaping_wrappers.py

```diff
@@ -49,20 +51,31 @@ def log_callback(self, logger):
         for rew_type, rews in self.ep_rew_dict.items():
             assert len(rews) == num_episodes
             means[rew_type] = sum(rews) / num_episodes
-            logger.logkv(f'ep{rew_type}mean', means[rew_type])
+            logger.logkv(f'shaping/ep{rew_type}mean', means[rew_type])
```

Comment:
Fine to change the tags to be hierarchical, but please then update the custom layout in `logger.py`.